### ISSUE TITLE
Allow hiding of algebraic description

### ISSUE THREAD

**Issue by pedritomelenas** [m2rash/audiofunctions-plus#23]
In the link provided, there should be an option to hide all the descriptions of the functions, so they cannot be visualized not read aloud by the screen reader.
